kong.conf.default

```diff
@@ -439,6 +429,57 @@
                                   # `text/html`, `application/json`, and
                                   # `application/xml`.
 
+#------------------------------------------------------------------------------
+# NGINX injected directives
+#------------------------------------------------------------------------------
+
+# Nginx directives can be dynamically injected in the runtime nginx.conf file
+# without requiring a custom Nginx configuration template.
+#
+# All configuration properties respecting the naming scheme
+# `nginx_<namespace>_<directive>` will result in `<directive>` being injected in
+# the Nginx configuration block corresponding to the property's `<namespace>`.
+# Example:
+#   `nginx_proxy_large_client_header_buffers = 8 24k`
+#
+#   Will inject the following directive in Kong's proxy `http {}` block:
```

Comment:
It would also be correct, unlike the current statement! Good catch, updating.

Side note:

There is intent to rename some of these prefixes to avoid such confusions:

- `nginx_proxy` -> `nginx_http_proxy` -> `http { server {8000 } }`
- `nginx_sproxy` -> `nginx_stream_proxy` -> `stream { server {} }`
- `nginx_admin` -> `nginx_http_admin` -> `http { server { 8001 } }`

There are `TODOs` in `conf_loader.lua` for this, but I kept it separate from this PR scope for now.